documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -1050,8 +1044,10 @@ sys/i386/i386/machdep.c:
 [.programlisting]
 ....
 sys/kern/subr_param.c:
-	hz = HZ;
+	hz = -1;
 	TUNABLE_INT_FETCH("kern.hz", &hz);
+	if (hz == -1)
+		hz = vm_guest > VM_GUEST_NO ? HZ_VM : HZ;
```

Comment:
Where are you getting this code from?

Reply:
From sys/kern/subr_param.c
Maybe you're suggesting I delete the first line? (hz = -1;)

Reply:
Nope, sorry. I was just confused because it's such a big change.

Reply:
Thank you!